### PR TITLE
update gogrid faq

### PR DESCRIPTION
**Pull request by stephamon**
JIRA 2460

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-9022125dab/how-to/). :bow:

File: content/managed-operations/gogrid-eol-faq.md [hunk 1]
```diff
@@ -5,40 +5,78 @@ title: GoGrid end of life FAQ
...
 
-Unfortunately, it is not possible to extend the End of Life date. Please contact us to review your migration options.
+It is not possible to extend the End of Life date. Please contact us to review
+your migration options.
 
 #### Will you continue to support GoGrid until the EOL?
 
-Yes, we will continue to support GoGrid and will be available to assist you with any issues you may have.
+Yes, we will continue to support GoGrid and will be available to assist you
+with any issues you may have.
```
Comment (RackerKate) [inline content/managed-operations/gogrid-eol-faq.md]: Suggest "...that you may have."

File: content/managed-operations/gogrid-eol-faq.md [hunk 2]
```diff
@@ -5,40 +5,78 @@ title: GoGrid end of life FAQ
...
+
+#### How can Rackspace help me decide on the best cloud infrastructure for my business?
```
Comment (RackerKate) [inline content/managed-operations/gogrid-eol-faq.md]: Suggest "...choose the best..."

File: content/managed-operations/gogrid-eol-faq.md [hunk 3]
```diff
@@ -5,40 +5,78 @@ title: GoGrid end of life FAQ
...
+As a leader in Managed Services for public cloud infrastructure, Rackspace
+delivers unbiased guidance on best-fit cloud solutions to organizations around
+the globe. We go beyond simple migration assistance and infrastructure
+management with multi-cloud managed services, professional services and managed
+application services to enable true digital transformation.
+
+#### What replacement solutions does Rackspace offer that I can migrate to?
+
+Rackspace has an extensive portfolio of cloud and single tenant solutions,
+including our Managed Public Clouds across AWS, Microsoft Azure, Google Cloud
```
Comment (RackerKate) [inline content/managed-operations/gogrid-eol-faq.md]: Suggest "...across Amazon Web Services (AWS), ..." and adding any appropriate registered trademark symbols after the names of those services.

Comment (stephamon) [reply content/managed-operations/gogrid-eol-faq.md]: Ah, I'll add all that to the intro paragraph, and expand AWS there.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5ae9a295d3/how-to/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-e66e151a15/how-to/)

**Pull request merged**